contract/data/mail_template.xml

```diff
@@ -91,6 +91,16 @@
     >
         <xpath expr="//t[@t-raw='message.body']" position="after">
             <t t-raw="0" />
+            <t t-if="record._name == 'contract.contract'">
```

Comment:
Shouldn't this be before t-raw? Have you checked it's working?

Reply:
I don't remember motive to set `t-raw=0` previously but with these changes all work fine, i have tested with some contract modification and mails and access_token have added to link button.